docs/checks.md

```diff
@@ -59,9 +59,8 @@ Allowed by Scorecards:
 Risk: `High` (vulnerable to intentional malicious code injection)  
 
```

Comment:
please ignore this file: some changes are due to a previous PR that did not keep the .yaml and .md in sync after updates.